syncmaster/schemas/v1/transfers/__init__.py

```diff
@@ -206,6 +207,10 @@ class ReadTransferSchema(BaseModel):
     transformations: list[Annotated[TransformationSchema, Field(..., discriminator="type")]] = Field(
         default_factory=list,
     )
+    resources: Resources = Field(
+        ...,
+        description="Spark resources",
```

Comment:
```suggestion
        description="Transfer resources",
```

The engine used by SyncMaster can be other than Spark in future